src/exercise/04.js

```diff
@@ -31,11 +36,21 @@ function Board() {
     // 💰 `squaresCopy[square] = nextValue`
     //
     // 🐨 set the squares to your copy
+    const nextValue = calculateNextValue(squares);
```

Comment:
This doesn't need to be calculated here, because it's being calculated above, and the value it's running against (`squares`) is defined in the upper scope, so it won't change inside of this function vs. out. So this line can be removed, an you can rely on what's being set in the upper scope.